patches/@guardian__support-dotcom-components@2.1.1.patch

```diff
@@ -0,0 +1,58 @@
+diff --git a/dist/shared/src/types/props/banner.d.ts b/dist/shared/src/types/props/banner.d.ts
```

Comment:
just for my own understanding/curiosity: what is this file and why/how does it get generated?

Reply:
It applies a apply a [`pnpm patch`](https://pnpm.io/cli/patch) to the dependency, which transforms it so the `tsc` step is successful. This will not be necessary if the upstream package is fixed in https://github.com/guardian/support-dotcom-components/pull/1119